tchannel/tracing.py

```diff
@@ -97,7 +97,11 @@ def span_in_context(self, span):
         :param span: an OpenTracing Span
         :return: ``StackContext``-based context manager
         """
-        return opentracing_instrumentation.span_in_stack_context(span)
+        if isinstance(opentracing.tracer.scope_manager,
+                      opentracing_instrumentation.request_context.TornadoScopeManager):  # noqa
```

Comment:
suspect you can make this pep8 friendly with

```
if isinstance(
    opentracing.tracer.scope_manager,
    opentracing_instrumentation.request_context.TornadoScopeManager,
):
    ...
```

Reply:
Nope still too long :)